oraseWithDBV20230105V1.py

```diff
@@ -0,0 +1,634 @@
+import os
+from datetime import datetime
+import mysql.connector
+
+
+
+def indexOf(line, deCautat):
+    poz = -1
+    try:
+        poz = line.index(deCautat)
+    except:
+        poz = -1
+    return poz
+
+
+class Oras():
+    def __init__(self, ordX=0, ordY=0, nume='', judet='', auto='', populatie=0, regiune=''):
+        self.ordX = ordX
+        self.ordY = ordY
+        self.nume = nume
+        self.judet = judet
+        self.auto = auto
+        self.populatie = populatie
+        self.regiune = regiune
+        
+    def __str__(self):
+        return self.ordX  +  self.ordY  +  self.nume  +  self.judet  +  self.auto  +  self.populatie  +   self.regiune
+        
+    def getOrdX(self):
+        return self.ordX
+    
+    def getOrdY(self):
+        return self.ordY
+    
+    def getNume(self):
+        return self.nume
+    
+    def getJudet(self):
+        return self.judet
+    
+    def getAuto(self):
+        return self.auto
+    
+    def getPopulatie(self):
+        return int(self.populatie)
+    
+    def getRegiune(self):
+        return self.regiune
+    
+    def setNume(self, nume):            
+        msg = self.validateNume(nume)
+        if msg == "ok":
+            if ord(nume[0]) >= ord('a') and ord(nume[0]) <= ord('z'):
+                nume = "" + chr(ord(nume[0])-32) + nume[1:]                
+            self.nume = nume
+        else:
+            raise TypeError(msg)
+            
+    def validateNume(self, nume):        
+        msg = "ok"
+        if len(nume) < 3:
+            msg = "Numele trebuie sa aiba cel putin 3 caractere"
+            
+        return msg
+    
+
+def loadFile(file):
+    f = open(file,'r')
+    arr = []
+    f.readline()
+    for line in f:
+        
+        latitudine = line[0:line.index(',')]
+        line = line[line.index(',')+1:]
+        
+        longitudine = line[0:line.index(',')]
+        line = line[line.index(',')+1:]
+         
+        oras = line[0:line.index(',')]
+        line = line[line.index(',')+1:]
+        
+        judet = line[0:line.index(',')]  
+        line = line[line.index(',')+1:]
+       
+        judetAuto = line[0:line.index(',')]
+        line = line[line.index(',')+1:]
+        
+        if line[0:line.index(',')] == '':
+            populatie = 0
+        else:
+            populatie = int(line[0:line.index(',')])
+            line = line[line.index(',')+1:]
+        
+        regiune = line[0:]
+       
+        arr.append(Oras(latitudine, longitudine, oras, judet, judetAuto, populatie, regiune))
+                   
+    f.close()
+    return arr
+
+mydb = mysql.connector.connect ( host="localhost", port="3306", user="agendatelefonica", password="root1234", database ="agendatelefonica" )
+
+22

```

Comment:
ce e acest 22?

Reply:
hahaha a ramas aiurea acolo, am copiat denumirea de meniu din loc in loc sa stiu ce sa scriu la OP = ADAUGA_ORAS gen si a ramas acolo, nu l am vaz sa l sterg